packages/bridge-ui-v2/src/components/Bridge/ProcessingFee/ProcessingFee.svelte

```diff
@@ -44,15 +44,15 @@
       // If so, let's switch to RECOMMENDED method
       selectedFeeMethod = ProcessingFeeMethod.RECOMMENDED;
     }
-
+    removeEscKeyListener();
```

Comment:
Not sure if we are already closing the modal on component unmounts, but I think it would be also a good idea to remove the listener onDestroy

Reply:
done